bundle/clientcache/client_cache.go

```diff
@@ -132,8 +131,8 @@ func SegmentCacheKey(checksum []byte) cache.ActionID {
 	return *((*cache.ActionID)(s.Sum(nil)))
 }
 
-func LabelCacheKey(label string) cache.ActionID {
+func SourceCacheKey(source string) cache.ActionID {
 	s := sha256.New()
-	_, _ = fmt.Fprintf(s, "cerbos:cloud:bundle:label=%s", label)
+	_, _ = fmt.Fprintf(s, "cerbos:cloud:bundle:source=%s", source)
```

Comment:
Since the key is changing anyway, should we replace `cloud` with `hub`?
```suggestion
	_, _ = fmt.Fprintf(s, "cerbos:hub:bundle:source=%s", source)
```